Automattic/newspack-plugin
# feat: consolidate campaigns into one tab

### All Submissions:






### Changes proposed in this Pull Request:

Consolidates campaign management into one tab, removes `Overlay` and `Inline` tabs.

Dependent PRs:
~https://github.com/Automattic/newspack-plugin/pull/764~
~https://github.com/Automattic/newspack-plugin/pull/767~

<img width="1420" alt="Screen Shot 2020-12-31 at 5 38 46 AM" src="https://user-images.githubusercontent.com/1477002/103407023-ed7c7b80-4b2a-11eb-9771-d7d6b2597ca1.png">

### How to test the changes in this Pull Request:

1. Observe there is only one campaign tab, verify it contains all campaigns and that filtering and campaign management works as expected.
2. Observe the placement is in the description. Options are `Inline|Overlay|Above header`

### Other information:
